Mathlib/NumberTheory/ArithmeticFunction.lean

```diff
@@ -1422,6 +1424,91 @@ theorem prod_eq_iff_prod_pow_moebius_eq_on_of_nonzero [CommGroupWithZero R]
 
 end SpecialFunctions
 
+section Sum
+
+theorem sum_zeta (N : ℕ) : ∑ n ∈ Icc 0 N, zeta n = N := by
```

Comment:
There are multiple ways to phrase this result, so I think it's best we specify this is a sum over `Icc`. 
```suggestion
theorem sum_Icc_zeta (N : ℕ) : ∑ n ∈ Icc 0 N, zeta n = N := by
```